src/providers/fault-store/fault-store.ts

```diff
@@ -151,15 +136,18 @@ export class FaultStoreProvider {
   }
 
   calculateTestResult() {
-    if (this.drivingFaultsNumber >= 16 || this.seriousFaultsNumber > 0 || this.dangerousFaultsNumber > 0) {
-      return this.testResult = TestResult.Fail;
+    if (
+      this.drivingFaultsNumber >= 16 ||
+      this.seriousFaultsNumber > 0 ||
+      this.dangerousFaultsNumber > 0
+    ) {
+      return (this.testResult = TestResult.Fail);
```

Comment:
Can we change this to two lines with an assignment and a return? It looks like a comparison done wrong!